zproject/urls.py

```diff
@@ -106,6 +106,10 @@
         {'PATCH': 'zerver.views.custom_profile_fields.update_realm_custom_profile_field',
          'DELETE': 'zerver.views.custom_profile_fields.delete_realm_custom_profile_field'}),
 
+    # realm/deactivate_realm -> zerver.views.deactivate_realm
+    url(r'^realm/deactivate_realm$', rest_dispatch,
+        {'DELETE': 'zerver.views.realm.deactivate_realm'}),
```

Comment:
this should be a POST, probably; you'd only want DELETE if the structure was `DELETE /realm/15`, which isn't what we have here.